.github/workflows/python-wheels-publish-test.yml

```
@@ -64,9 +64,9 @@ jobs:
           CIBW_ENVIRONMENT: OPENEXR_RELEASE_CANDIDATE_TAG="${{ github.ref_name }}"
 
       - name: Upload artifact
-        uses: actions/upload-artifact@ff15f0306b3f739f7b6fd43fb5d26cd321bd4de5 # v3.2.1
+        uses: actions/upload-artifact@v4
         with:
-          name: wheels-${{ matrix.os }}
+          name: wheels-${{ matrix.os }}-${{ env.INSTALL_MANIFEST }}
```

`-${{ env.INSTALL_MANIFEST }}` should definitely not get appended to the name, that was appropriate only for the install manifests in `ci_steps.yml`.

Leave this line as it was for now. This workflow gets run only during the release process so it's a little hard to test, but I'll review it before the next release to make sure it does the right thing.
```suggestion
          name: wheels-${{ matrix.os }}
```